app/api/settings/specs/settings.spec.js

```diff
@@ -190,6 +190,11 @@ describe('settings', () => {
       const values = await settings.get();
       expect(values.publicFormDestination).not.toBeDefined();
     });
+
+    it('should return default values', async () => {
+      const values = await settings.get();
+      expect(values.mapTilerKey).toEqual('QiI1BlAJNMmZagsX5qp7');
+    });
```

Comment:
This test covers only the scenario of returning the default mapTilerKey.  But it does not test the fact that if the DB already contains such a key, it should return that.  Please, try to approach this as TDD: write the test that expects that, if the DB already has a value, to be that value instead of the default.  See the test fail, implement only the code required to make that small part fail.

Reply:
Thank you Rafa, that's true, the initial code covered this case without a test.
I've added a test that saves a mapTilerKey and then checks if get returns this one.